code-challenges.test.js

```diff
@@ -17,35 +17,81 @@
 const people = [
   { name: "ford prefect", occupation: "a hitchhiker" },
   { name: "zaphod beeblebrox", occupation: "president of the galaxy" },
-  { name: "arthur dent", occupation: "a radio employee" }
-]
+  { name: "arthur dent", occupation: "a radio employee" },
+];
 // Expected output: ["Ford Prefect is a hitchhiker.", "Zaphod Beeblebrox is president of the galaxy.", "Arthur Dent is a radio employee."]
 
+// a describe method that lists the name of the function OR naming of the particular test.
+describe("capName", () => {
+  // a test/it method, nested within the describe block, that in plain words, describes that the function does.
+  it("capName will capitalize the name and join the occupation and return the sentence in an array.", () => {
+    //an expect method, nested within the test block, calling on the hello() function, followed by the .toEqual() matcher that checks the expected output of the function return.
+    expect(
+      capName(string.charAt(0).toUpperCase() + string.slice(1).join)
+    ).toEqual("Ford Prefect is a hitchhiker.");
```

Comment:
Tests are just a place to provide your expected input and output. You wouldn't typically write logic in your test.